youtube.js

```diff
@@ -2,7 +2,7 @@ var showOnlyFirstName;
 
 var highlightWords = [];
 
-$("body").unbind("click").on("click", "yt-live-chat-text-message-renderer,yt-live-chat-paid-message-renderer,yt-live-chat-membership-item-renderer,yt-live-chat-paid-sticker-renderer", function () {
+window.location.href.indexOf('youtube') > 0 && $("body").unbind("click").on("click", "yt-live-chat-text-message-renderer,yt-live-chat-paid-message-renderer,yt-live-chat-membership-item-renderer,yt-live-chat-paid-sticker-renderer", function () {
```

Comment:
This is the hack that makes this work on multiple URLs. It's cheesy but seems to work OK. 

If a few things were factored out, it wouldn't be unreasonable to put the entire youtube.js inside a conditional.